failed to build cc-runtime on ARM

# Description of problem

failed to build cc-runtime on ARM

# Expected result

cc-runtime binary

# Actual result

Makefile:35: arch/arm64-options.mk: No such file or directory
make: *** No rule to make target 'arch/arm64-options.mk'.  Stop.

---
configuration file should have `qemu-system-aarch64` as qemu command and `kvm` as machine accelerator

--- a/arch/arm64-options.mk
+++ b/arch/arm64-options.mk
@@ -1,0 +1,21 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ARM 64 settings
+
+MACHINETYPE := virt
+KERNELPARAMS :=
+MACHINEACCELERATORS :=
+
+QEMUCMD := qemu-system-aarch64

